cl_dll/StudioModelRenderer.h

```diff
@@ -101,6 +101,9 @@ class CStudioModelRenderer
 	// Process movement of player
 	virtual void StudioProcessGait ( entity_state_t *pplayer );
 
+	// Set the OpenGL
```

Comment:
Seems like you accidentally a comment end

Reply:
Haha, yeah :smile: